### PR TITLE
Additional fixes to integration tests

### PR DESCRIPTION
**Pull request by hickeng**
This builds on #8047 and #8063 to address some outstanding issues left by
that work. In the case of the hetrogeneous and NFS tests, it is test
updates necessitated by the prior Nimbus-Util.robot updates.
In the case of the VSAN tests it is addressing removal of a non-existent
path and switching to use the related testbed spec.

Towards #8067

### CODE REVIEW

File: tests/resources/Nimbus-Util.robot [hunk 1]
```diff
@@ -571,3 +572,17 @@ Get Name of First Local Storage For Host
     ${datastores}=  Run  govc host.info -host ${host} -json | jq -r '.HostSystems[].Config.FileSystemVolume.MountInfo[].Volume | select (.Type\=\="VMFS") | select (.Local\=\=true) | .Name'
     @{datastores}=  Split To Lines  ${datastores}
     [Return]  @{datastores}[0]
+
+# Simple wrapper to Wait Until Keyword Succeeds that allows callers to:
+# * use default retry count and delay
+# * specify specific retry counts and delays
+# * robot framework executor to override both the above via the following environment variables:
```
Comment (zjs) [inline tests/resources/Nimbus-Util.robot]: nit: This documentation reads a bit awkwardly. ("Simple wrapper to Wait Until Keyword Succeeds that allows callers to ... robot framework executor to override ...")

File: tests/resources/Nimbus-Util.robot [hunk 2]
```diff
@@ -571,3 +572,17 @@ Get Name of First Local Storage For Host
...
+#   NIMBUS_RETRY_ATTEMPTS
+#   NIMBUS_RETRY_DELAY
+Nimbus Suite Setup
+    [Arguments]  ${keyword}  ${attempts}=1x  ${delay}=1m  @{varargs}
+
+    ${useAttempts}=  Get Environment Variable  NIMBUS_RETRY_ATTEMPTS  ${attempts}
+    ${useDelay}=     Get Environment Variable  NIMBUS_RETRY_DELAY     ${delay}
+
+    Wait Until Keyword Succeeds  ${useAttempts}  ${useDelay}  ${keyword}  @{varargs}
```
Comment (zjs) [inline tests/resources/Nimbus-Util.robot]: nit: missing newline at end of file

File: tests/resources/nimbus-testbeds/vic-vsan.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+require ‘testframeworks/testng/testng’
```
Comment (zjs) [inline tests/resources/nimbus-testbeds/vic-vsan.rb]: Stupid question: is this the correct type of quotation mark for this context?

Comment (hickeng) [reply tests/resources/nimbus-testbeds/vic-vsan.rb]: I've no idea - I was given it by someone from the channel. I've dropped this file entirely in favour of a simpler construction.

**Pull request merged**